CONTRIBUTING.md

```diff
@@ -96,6 +96,50 @@ Closes #123
 
 In this case 123 is the corresponding issue number.
 
+### When End-To-End Tests Fail
+
+Every pull request triggers a full run of testing in the CI system.
+The failures reported by the code style checker (aka the "linter") and the unit tests are usually
+clear and easy to fix (and can be avoided by running `npm t` locally before creating a commit).
+But when an integration, or e2e test, fails, it's sometimes useful to consult the log files
+for the run.
+
+These can be found by first going to the github page for the PR (for an example, find an open PR at https://github.com/rancher-sandbox/rancher-desktop/pulls?q=is%3Aopen+is%3Apr),
+then look for the list of checks, find the `E2E/Integration Tests` entry, and click on the `Details` link for the first view.
+
+This takes you to a more detailed view of the test, and which tests failed, but the log files are a few more clicks away.
+
+First, scroll to the bottom and click on the link that reads `View more details on Cirrus CI`.
+
+On that page, click on the `Debug` icon near the top-right of the view.
+
+Scroll to the bottom of the text view, where there should be a section labelled `Artifacts`, with the text `playwright`.
+
+Clicking on the `playwright` bar should download a file called `playwright.zip` to your download directory.
```

Comment:
It's a little more… annoying than that.
![image](https://user-images.githubusercontent.com/3977982/182502992-3743e7a3-8988-461c-bd7b-4370b34522c9.png)
If you click the download icon ⬇ it downloads `playwright.zip`; but if you click outside of that icon but in that row it expands to let you browse the zip file directly:
![image](https://user-images.githubusercontent.com/3977982/182503028-0d5d4331-87e7-49bc-81f0-f3fe1f552628.png)
For our uses I'd recommend the list view over the tree view (since the tree view is more like directory view, and pretty useless unless there's too many files):
![image](https://user-images.githubusercontent.com/3977982/182503139-3ed9b41c-6779-404f-9039-bedf1ad157e2.png)

Reply:
Added screenshots with highlighted sections.